README.md

```diff
@@ -22,9 +22,12 @@ sudo apt-get install texlive-xetex
 # initaite a new contract
 claw new john examples/templates/consultant.md
 
+# change into the newly generated directory
+cd john
+
 # edit the params
-vim john/params.toml
+vim params.toml
 
 # compile the markdown and output a final pdf using pandoc
-claw compile --output pdf john
+claw compile --output pdf
```

Comment:
it no longer expects an arg because the file path is different now. that was the point of these changes. it works. just the formatting is messed up. see next comment. will fix and push in the morning.